build.sh

```diff
@@ -176,7 +176,7 @@ BuildLibevent()
     if [ $? -ne 0 ];then
         exit 1
     fi    
-    make
+    make -j
```

Comment:
It's better to use `make -j ${NUM_CORE}`,
and let get `NUM_CORE` automatically.